File: FarNet/Plugins.NET/Hello.CS/bin/Hello.cs
using FarManager;
public class HelloCS : BasePlugin
{
	void item_OnOpen(object sender, PluginMenuEventArgs e)
	{
		Far.Msg("Hello, world from c#", "Far.NET");
	}
	void sayHello(object sender, ExecutingEventArgs e)
	{
		Far.Msg("C#: Hello, " + e.Command);
	}
	override public void Connect()
	{
		Far.RegisterPluginsMenuItem("Hello c#", item_OnOpen);
		Far.RegisterPrefix("hellocs", sayHello);
	}
}

File: tags/3.3.33/Plugins.NET/Hello.VB.NET/bin/Hello.vb
Imports FarManager
Public Class HelloVB
Inherits BasePlugin
	Sub item_OnOpen(ByVal sender As Object, ByVal e As PluginMenuEventArgs)
		Far.Msg("Hello, world from vb.net", "Far.NET")
	End Sub

	Sub sayHello(ByVal sender As Object, ByVal e As ExecutingEventArgs)
		Far.Msg("VB.NET: Hello, " + e.Command)
	End Sub

	Public Overloads Overrides Sub Connect()
		Far.RegisterPluginsMenuItem("Hello vbnet", AddressOf item_OnOpen)
		Far.RegisterPrefix("hellovb", AddressOf sayHello)
	End Sub
End Class
